### ISSUE TITLE
buildings highlighted are only ways, not multipolygon relations

### ISSUE THREAD

**Issue by MarcoMinghini** [hotosm/osm-analytics#34]
Buildings found on the map are just the **ways** having building=*, while **relations** having building=* and type=multipolygon (typically buildings having an inner courtyard) are not shown (example: http://www.openstreetmap.org/relation/1120279)

**Comment by MarcoMinghini** [comment]
Sorry, I've just read this in the README. However I think this is an important feature to add so I hope to see it in the future. Thanks!

**Comment by tyrasd** [comment]
yeah, it's one of the current [limitations](https://github.com/hotosm/osm-analytics#multipolygons). It would be possible to add without almost any effort to osm-analytics, but requires having the multipolygon geometries is the osm-qa-tiles. And from what I can see, it should be possible to include them as the producing code seems to already [support](https://github.com/mapbox/minjur/issues/8) it. On the other hand I'm not 100% sure if this is going to land in osm-qa-tiles anytime soon… //cc @aaronlidman

**Comment by aaronlidman** [comment]
We're looking into adding multipolygons, which would support most buildings, into the underlying osm-qa-tiles. While already technically possible, it greatly increases processing time and output size, so we're looking into other ways of doing it.

Here are our initial results:

command | minjur time | geojson size | tippecanoe time | mbtiles size | total time
----- | ----- | ----- | ----- | ----- | -----
minjur | 1h14m | 210 GB | 2h22m | 26 GB (25 gzip) | 3h36m
minjur-mp | 2h34m | 329 GB | 3h57m | 64 GB (57 gzip) | 6h31m
